packages/pds/src/db/migrations/20230508T204041032Z-disable-account-invites.ts

```diff
@@ -0,0 +1,15 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('user_account')
+    .addColumn('invitesDisabled', 'int2', (col) => col.defaultTo(0))
```

Comment:
Might recommend marking this as `notNull()` since it has a default :)

Reply:
ah good call 👍